src/data-access/DevItemSearcher.ts

```diff
@@ -42,7 +42,7 @@ export default class DevItemSearcher implements ItemSearcher {
 			origin: '*',
 		} ) ).then( ( r ) => r.json() ) as WbSearchEntitiesResponse;
 
-		return processResponse( response, 'en' );
```

Comment:
Why is this redundant now?

Reply:
Because `processResponse()` no longer has a `defaultLanguageCode` parameter; it was only needed to provide a language for the label and description when it wasn’t included in the response, but now we always have the language in the response itself.